sqlglot/dialects/presto.py

```diff
@@ -317,6 +317,7 @@ class Generator(generator.Generator):
             exp.DataType.Type.STRUCT: "ROW",
             exp.DataType.Type.DATETIME: "TIMESTAMP",
             exp.DataType.Type.DATETIME64: "TIMESTAMP",
+            exp.DataType.Type.HLLSKETCH: "HYPERLOGLOG",
```

Comment:
We also need to map the hyperloglog keyword to the hllsketch token right?

Reply:
ah, basically add: "HYPERLOGLOG": TokenType.HLLSKETCH 
 in the keywords? map, so we're mapping through this. Right?